pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -852,22 +864,44 @@ def get_instancename(context, instancename, options, ns_names=None):
         Command-specific options from the command line (including --key and
         --namespace if specified).
 
-      ns_names (list of :term:`string` or None)
-        If None, this is a command with a single namespace.
-        If a list, this is a command with multiple namespaces
+      default_all_ns (:class:`py:bool`)
+        If True, the default is to return all namespaces.  Otherwise the
+        default is to return None
+
+      ns_names (list of :term:`string`, :term:`string`, or None)
```

Comment:
You removed the `ns_names` parameter from the method.

Reply:
Modified to create the ns_names parameter within the method so this parameter reference was removed.